src/components/AppSwitch.js

```diff
@@ -73,7 +73,13 @@ class AppSwitch extends React.Component<LoadingProps, {}> {
 
       if (jwt) {
         log.debug('New account, not verified, or did not finish signup', jwt)
-        this.props.navigation.navigate('Auth')
+
+        if (this.props.store.get('destinationPath') !== '') {
+          this.props.navigation.navigate(JSON.parse(this.props.store.get('destinationPath')))
```

Comment:
I prefer to use `const { navigation, store } = this.props` at the beginning of the method, so it's less text, but I think it's just a nice to have

Reply:
done, thanks!